Source/WebKit/UIProcess/WebsiteData/WebsiteDataStore.cpp

```diff
@@ -1907,6 +1907,13 @@ WebsiteDataStoreParameters WebsiteDataStore::parameters()
 #if !HAVE(NSURLSESSION_WEBSOCKET)
     networkSessionParameters.shouldAcceptInsecureCertificatesForWebSockets = m_configuration->shouldAcceptInsecureCertificatesForWebSockets();
 #endif
+    networkSessionParameters.isBlobRegistryTopOriginPartitioningEnabled = [this](){
+        bool isBlobPartitioningEnabled { false };
+        WebProcessProxy::forWebPagesWithDataStore(*this, [&isBlobPartitioningEnabled](const WebPageProxy& page) {
+            isBlobPartitioningEnabled |= page.preferences().blobRegistryTopOriginPartitioningEnabled();
```

Comment:
Should use `||=` since we're not dealing with bits here.

That said, this should call `WebProcessProxy::isBlobRegistryPartitioningEnabled(sessionID)` to avoid code duplication.